Unreal Engine plugin documentation update (#6350)
getsentry/sentry-docs: src/platform-includes/enriching-events/set-tag/unreal.mdx

![Set tag](unreal_set_tag.png)

Alternatively, this configuration cab be provided to the crash reporter [during initialization](/platforms/unreal/setup-crashreporter/#configure-attributes).

> Commonly used values like `Engine version`, `Configuration`, `Target Type`, etc. can be promoted to tags by using the Sentry configuration window. Navigate to the editor's menu **Project Settings > Plugins > Sentry > Misc > Promote values to tags** and select which values should be automatically added to captured events as tags.
> 

Reviewer: ```suggestion
Commonly used values including `Engine version`, `Configuration`, and `Target Type`, can be promoted to tags via Sentry's configuration window by navigating to the editor's menu **Project Settings > Plugins > Sentry > Misc > Promote values to tags** and selecting the values that should be automatically added to captured events as tags.

```
